app/assets/javascripts/views/file_uploader_view.js.coffee

```diff
@@ -1,18 +1,20 @@
-class ELMO.Views.MediaUploaderView extends ELMO.Views.ApplicationView
+class ELMO.Views.FileUploaderView extends ELMO.Views.ApplicationView
```

Comment:
Can you refactor this class to use `camelCase` variables throughout? I want us to move us in that direction—right now there is a mixture. The serializer lib we use can do this so that incoming data is in camelCase also (but that's not used here, just for future reference). You'll have to refactor the instantiation call in the template slightly as well.

Reply:
(Remember to use Cmd+D to select multiple occurrences. It will go quickly that way.)

Reply:
Please also add a class comment.